AppSettingsManagerBff/AppSettingsManagerBff.Domain/ApiRepositories/IUserRepository.cs

```diff
@@ -5,5 +5,5 @@ namespace AppSettingsManagerBff.Domain.ApiRepositories;
 public interface IUserRepository
 {
     Task<ApiBaseUser> GetUser(string userId);
-    Task<ApiBaseUser> CreateUser(string userId, string password);
+    Task<ApiBaseUser> CreateUser(string userId, string password); 
```

Comment:
Need to add UpdateUser and DeleteUser methods too that match what's in HttpUserRepository. The controller class instantiates IUserRepository so any method that's not listed here won't be available in the controller